documentation/guide-jpa-query.asciidoc

```diff
@@ -49,14 +101,21 @@ Here is an example from our sample application:
     if ((name != null) && (!name.isEmpty())) {
       query.where(dish.name.eq(name));
     }
+    query.orderBy(dish.price.asc(), dish.name.asc());
     return query.fetch();
   }
 ----
 
-In this example we use the so called Q-types (`QDishEntity`). These are classes generated at build time by the QueryDSL annotation processor from entity classes. The Q-type classes can be used as static types representative of the original entity class.
+In this example, we use the so called Q-types (`QDishEntity`). These are classes generated at build time by the QueryDSL annotation processor from entity classes. The Q-type classes can be used as static types representative of the original entity class.
+
+The `query.from(dish)` method call defines the query source, in this case the `dish` table. The `where` call defines a filter. For example, The first call uses the `goe` operator to filter out any dishes that are not greater or equal to the minimal price. Further operators can be found https://querydsl.com/static/querydsl/4.4.0/apidocs/com/querydsl/core/types/dsl/ComparableExpression.html[here].
+
+The `orderBy` method is used to sort the query results according to certain criteria. Here, we sort the results first by their price and then by their name, both in ascending order. To sort in descending order, use `.desc()`. To partition query results into groups of rows, see the https://querydsl.com/static/querydsl/4.1.3/reference/html_single/#d0e377[groupBy] method.
```

Comment:
You are using two links to different versions of QueryDsl (4.1.3 and 4.4.0). I would recommend to use the same version. You can also replace the version with "latest". Then the link always points to the latest documentation

https://querydsl.com/static/querydsl/latest/apidocs/com/querydsl/core/types/dsl/ComparableExpression.html
https://querydsl.com/static/querydsl/latest/reference/html_single/#d0e392